packages/wallets/ONTO/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@solana/wallet-adapter-ONTO",
+    "version": "0.4.0",
+    "author": "Solana Maintainers <maintainers@solana.foundation>",
+    "repository": "https://github.com/solana-labs/wallet-adapter",
+    "license": "Apache-2.0",
+    "type": "module",
+    "sideEffects": false,
+    "main": "lib/cjs/index.js",
+    "module": "lib/esm/index.js",
+    "types": "lib/esm/index.d.ts",
+    "exports": {
+        "import": "./lib/esm/index.js",
+        "require": "./lib/cjs/index.js"
+    },
+    "files": [
+        "lib",
+        "src",
+        "LICENSE"
+    ],
+    "publishConfig": {
+        "access": "public"
+    },
+    "scripts": {
+        "clean": "shx rm -rf lib/*",
+        "build": "yarn clean && tsc -p tsconfig.json && tsc -p tsconfig-cjs.json",
```

Comment:
This needs to be updated to use `tsc-esm` to produce .mjs files, check out how the other wallet packages do this.

Reply:
have update

Reply:
This hasn't been updated. Please check how this is done in the other packages.